packages/cli/src/options/doppelgangerOptions.ts

```diff
@@ -0,0 +1,13 @@
+import {ICliCommandOptions} from "../util/index.js";
+
+type DoppelgangerOps = {
+  doppelgangerProtectionEnabled?: boolean;
+};
+
+export const doppelgangerOptions: ICliCommandOptions<DoppelgangerOps> = {
+  doppelgangerProtectionEnabled: {
+    description: "Enables Doppelganger protection",
+    default: false,
```

Comment:
should this be enabled by default?

Reply:
I don't believe Lighthouse enables it by default. They have an entire page dedicated to understanding the intricacies of it and the user must enable it themselves with the flag. I would assume the same for us too?
https://lighthouse-book.sigmaprime.io/validator-doppelganger.html